docs/install-include.rst

```diff
@@ -164,8 +164,13 @@ Select your preferences below and run the corresponding install commands:
 AutoGluon is modularized into `sub-modules <https://packaging.python.org/guides/packaging-namespace-packages/>`_ specialized for tabular, text, or image data. You can reduce the number of dependencies required by solely installing a specific sub-module via:  `python3 -m pip install <submodule>`, where `<submodule>` may be one of the following options:
 
 - `autogluon.tabular` - only functionality for tabular data (TabularPredictor)
+    - The default installation of `autogluon.tabular` standalone is a skeleton installation.
+    - Install via `pip install autogluon.tabular[all]` to get the same installation as via `pip install autogluon`
+    - Available optional dependencies: `lightgbm,catboost,xgboost,fastai`. These are included in `all`.
```

Comment:
May be good to provide example for novices like:

To run autogluon.tabular with only the optional LightGBM and CatBoost models for example, you can do:
`pip install autogluon.tabular[lightgbm,catboost]`

Reply:
added